frontend/src/entities/configuration/system-banner-configuration.tsx

```diff
@@ -0,0 +1,120 @@
+/**
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License").
+ You may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import {
+    Container,
+    FormField,
+    Grid,
+    Header,
+    Input,
+    SpaceBetween,
+    Toggle
+} from '@cloudscape-design/components';
+import React from 'react';
+
+export type SystemBannerConfigurationProps = {
+    setFields: (any) => void;
+    textColor: string;
+    backgroundColor: string;
+    text: string;
+    isEnabled: boolean;
+    touchFields: (any) => void;
```

Comment:
We also have a type definition for [touchFields](https://github.com/awslabs/mlspace/blob/main/frontend/src/shared/validation/index.ts#L134)